worker/protos/ffmpeg_worker.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+
+service FFmpeg {
+  rpc transcode(Request) returns (stream Log) {}
+}
+
+message Log {
+  string text = 1;
+}
+
+message Request {
```

Comment:
Name them  FfmpegRequest and FfmpegResponse . 
You can follow the conventions described here :
https://cloud.google.com/apis/design/naming_convention?hl=en